apps/root/src/pages/earn/components/all-strategies-table/index.tsx

```diff
@@ -147,7 +149,7 @@ const AllStrategiesTableBodySkeleton = () => (
 );
 
 const createRow = (strategy: Strategy) => (
-  <TableRow>
+  <TableRow key={`${strategy.id}-${strategy.guardian?.id}`}>
```

Comment:
stategy id is already going to be unique :) since it refers to the farm with the specific guardian